src/SFA.DAS.ApplyService.InternalApi/Infrastructure/AssessorServiceApiClient.cs

```diff
@@ -16,27 +16,28 @@
 
 namespace SFA.DAS.ApplyService.InternalApi.Infrastructure
 {
-    public class AssessorServiceApiClient
+    public class AssessorServiceApiClient : ApiClientBase<AssessorServiceApiClient>
     {
-        private readonly HttpClient _client;
-        private readonly ILogger<AssessorServiceApiClient> _logger;
         private readonly IApplyConfig _config;
         private readonly IHostingEnvironment _hostingEnvironment;
 
-        public AssessorServiceApiClient(HttpClient client, ILogger<AssessorServiceApiClient> logger,
-            IConfigurationService configurationService, IHostingEnvironment hostingEnvironment)
+        public AssessorServiceApiClient(ILogger<AssessorServiceApiClient> logger, IConfigurationService configurationService, IHostingEnvironment hostingEnvironment) : base(logger)
         {
-            _client = client;
-            _logger = logger;
             _config = configurationService.GetConfig().Result;
             _hostingEnvironment = hostingEnvironment;
+
+            if (_httpClient.BaseAddress == null)
+            {
+                _httpClient.BaseAddress = new Uri(_config.AssessorServiceApiAuthentication.ApiBaseAddress);
+            }
+            _httpClient.DefaultRequestHeaders.Authorization = new AuthenticationHeaderValue("Bearer", GetToken());
         }
 
         public async Task<IEnumerable<Types.OrganisationSearchResult>> SearchOrgansiation(string searchTerm)
         {
             _logger.LogInformation($"Searching EPAO Register. Search Term: {searchTerm}");
             var apiResponse =
-                await Get<IEnumerable<OrganisationSummary>>($"/api/ao/assessment-organisations/search/{searchTerm}");
+                await Get<List<OrganisationSummary>>($"/api/ao/assessment-organisations/search/{searchTerm}");
```

Comment:
There's a lot of this replacing IEnumerable with List, and the reason isn;'t clear in the code.  What's it for?

Reply:
Needed for generics to work correctly

Reply:
ok